tests/test_check_parallel.py

```diff
@@ -231,6 +233,22 @@ def test_worker_check_single_file_no_checkers(self) -> None:
         assert stats.statement == 18
         assert stats.warning == 0
 
+    def test_linter_with_plugins_is_pickleable(self) -> None:
+        """
+        The linter needs to be pickle-able in order to be passed between workers
+        """
+        dummy_plugin_path = Path(__file__).parent / "regrtest_data" / "dummy_plugin"
+        dummy_plugin_path_str = str(dummy_plugin_path.absolute())
+        sys.path.append(dummy_plugin_path_str)
+        linter = PyLinter(reporter=Reporter())
+        print(linter._dynamic_plugins)
```

Comment:
```suggestion
```

Reply:
I think something has gone weird with my precommit here because I definitely deleted that print call after using it to confirm that the plugin was definitely loading.
I did have some trouble with the hooks and my terminal so perhaps I've ended up committing some awful hybrid. 
Sorry for that, poor performance on my part, I'll review and push up again this morning.

Reply:
I don't know if this was the issue but pre-commit may make changes locally and you need to add those changes before committing again.

Reply:
Yes, I am familiar with it — I use it myself in many projects. It reinstalled the environments twice last night, and hung on one commit, though that might have been the terminal application I was using. I thought I'd come out the other end fine, had no unstaged changes, but didn't think to double-check the actual code I'd committed! Lesson learned in un-deserved confidence there.
In any case, it's back to the drawing board: I fixed what must have gone wrong and now have the test as I originally imagined, but @cdce8p is still spot on that the test is pointless! Test passes even without the fix, modules loaded or no. I'll find another way to test it.

Reply:
Ok, this was entirely a slip-up on my part — I did not take enough care when checking what I was actually testing. Not the first time I've made that mistake in this codebase (and probably many others 😱). Thanks for your patience as always! This is now resolved.